src/main/webapp/app/shared/modal/modify-cancer-type-modal.store.ts

```diff
@@ -109,3 +109,13 @@ export class ModifyCancerTypeModalStore {
     this.isRetryButtonClicked = false;
   }
 }
+
+// Once https://github.com/oncokb/oncokb-pipeline/issues/413 is fixed, we can remove this function.
+// Then we should complete this ticket: https://github.com/oncokb/oncokb-pipeline/issues/494 to backfill
+// all place where code == "" in firebase.
+const removeCancerTypeCode = (cancerTypeOption: CancerTypeSelectOption) => {
+  if (cancerTypeOption.level < 0) {
+    cancerTypeOption.code = null;
```

Comment:
Do we really need to do this?

Reply:
Before that ticket is resolved, yes.
Currently for data release, I have to manually remove all code for level < 0.